authenticity-collector/src/sludata/sludata.controller.ts

```diff
@@ -1,19 +1,56 @@
 import { Body, Controller, Post, UsePipes, ValidationPipe, Res } from '@nestjs/common';
 import { Response } from 'express';
 import { SluDataDto } from './model/SluDataDto';
-import { SludataService } from './sludata.service';
+import { SluDataService } from './sludata.service';
+import { ApiTags, ApiOperation, ApiBody, ApiResponse, ApiNotFoundResponse, ApiInternalServerErrorResponse } from '@nestjs/swagger';
 
+@ApiTags('slu-data')
 @Controller('/api/v1/authenticity/data')
-export class SludataController {
-	constructor(private sludataService: SludataService) {}
+export class SluDataController {
+	constructor(private sluDataService: SluDataService) {}
 
 	@Post()
+	@ApiResponse({
+		status: 201,
+		description: 'Sending authentication proof was successful',
+		schema: {
+			example: {
+				payload: {
```

Comment:
the response is ChannelData from the channel. Have a look at ChannelDataSchema of the ChannelClient.